exo-deploy/src/docker/docker-hub.ls

```diff
@@ -40,7 +40,8 @@ class DockerHub
         service-config = require path.join(process.cwd!, config.location, 'service.yml')
         names.push do
           author: service-config.author
-          name: path.basename config.location
+          name: service-config.title |> (.replace /\s/g, '-')
```

Comment:
Use https://www.npmjs.com/package/kebab-case to translate the string properly into kebab-case.